C++/syntax_test_c.c

```diff
@@ -589,9 +589,17 @@ int foo(int val, float val2[])
     if (val == -1) {
 /*  ^^ keyword.control */
 /*                 ^ meta.block meta.block punctuation.section.block.begin */
+#elifdef BAR
+ /* ^ meta.block meta.block */
```

Comment:
We probably want to make sure, those new keywords are scoped `keyword` as this is the major change of this PR.

```suggestion
 /* ^ meta.block meta.block keyword.control.import */
```

Reply:
> We probably want to make sure, those new keywords are scoped `keyword` as this is the major change of this PR.

Well that's the strange thing. When I tried to do that, it failed the assertion when I ran the syntax tests from within Sublime Text 4180. `keyword.control.import` would work for `#elif` but none of my new additions and I can't determine why.

Reply:
Well, checked out your branch and all scopes are applied as expected. Looks all good.

Have you symlinked C++ and Objective-C++ folders into your ST's _Packages_ directory, so ST can see the changes?

To not bother you with details, I've updated tests for you.

Reply:
I re-ran the syntax tests from Sublime Text and got these from your recent changes:
```Packages/User/C++/syntax_test_c.c:592:1
error: scope does not match
592 | #elifdef BAR
593 |  /* <- keyword.control.import */
    |  ^ this location did not match
actual:
    | ^^^^^^^^^ source.c meta.function.c meta.block.c meta.block.c 

Packages/User/C++/syntax_test_c.c:596:1
error: scope does not match
596 | #elifndef BAZ
597 |  /* <- keyword.control.import */
    |  ^ this location did not match
actual:
    | ^^^^^^^^^^ source.c meta.function.c meta.block.c meta.block.c meta.block.c 

FAILED: 2 of 674 assertions in 1 files failed
[Finished]```

Reply:
I see. So that's the issue. I thought it was rather strange. It's working now.